app/indexers/data_indexer.rb

```diff
@@ -18,10 +18,10 @@ def to_solr
       solr_doc['modified_latest_dttsi'] = modified_latest
       solr_doc['created_at_dttsi'] = created_at
 
-      # These are required as long as dor-services-app uses ActiveFedora for querying:
-      solr_doc['has_model_ssim'] = legacy_model
```

Comment:
Any reason to leave `#legacy_model` hanging around below? It looks like this was the only call to that method.

Reply:
Thanks for catching this. I've removed it.